### PR TITLE
Describe how floating tags are chosen

### PR DESCRIPTION
**Pull request by twaugh**
Signed-off-by: Tim Waugh <twaugh@redhat.com>

### CODE REVIEW

File: docs/users.rst [hunk 1]
```diff
@@ -398,11 +398,17 @@ Dockerfile, the tags used within the repository are:
...
+If the ``tags`` value is set in container.yaml, those tags are applied
+to the image as floating tags.
+
+If ``tags`` is not used in container.yaml, the following tags are
+applied:
 - ``{version}`` (the ``version`` label)
-- ``{version}-{release}`` (the ``version`` and ``release`` labels together)
-- any additional tags configured in the git repository (named in the
-  ``additional-tags`` file)
+- ``latest``
```
Comment (MartinBasti) [inline docs/users.rst]: This is not showing a list with bullets but inline on single line in GH view. (GH issue?)
IMHO it needs extra newline after `applied:`

Comment (twaugh) [reply docs/users.rst]: Thanks, should be fixed now.

File: docs/users.rst [hunk 2]
```diff
@@ -398,11 +398,18 @@ Dockerfile, the tags used within the repository are:
...
+
+If ``tags`` is not used in container.yaml, the following tags are
+applied:
+
 - ``{version}`` (the ``version`` label)
-- ``{version}-{release}`` (the ``version`` and ``release`` labels together)
-- any additional tags configured in the git repository (named in the
-  ``additional-tags`` file)
+- ``latest``
+- any additional tags named in the ``additional-tags`` file
```
Comment (ktdreyer) [inline docs/users.rst]: Tim, I was wondering about this `additional-tags` file. Is this a legacy thing that users should not use anymore?

Comment (twaugh) [reply docs/users.rst]: It's certainly less flexible. We haven't deprecated it yet though.

Comment (twaugh) [reply docs/users.rst]: Deprecation tracked in OSBS-7698.

**Pull request merged**